packages/og-utils/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@repo/og-utils",
+  "version": "0.0.0",
+  "private": true,
+  "exports": {
+    ".": "./src/index.ts"
+  },
+
+  "typesVersions": {
```

Comment:
this makes importing work but IIRC there is a better way to do this now.

cc @anthonyshew

Reply:
You shouldn't need `typesVersions` at all. 🤔

Reply:
removed it in https://github.com/typehero/typehero/pull/1545/commits/35b50b4d01830603f006a402ae0863b72acdba6a lets see 🙏

Reply:
@anthonyshew i removed it 💀
![image](https://github.com/user-attachments/assets/b90f2429-dfb1-44fc-9eb7-2d81198e6a3f)

Reply:
we can revisit this but i reverted it so we can ship this and figure out later.

Reply:
Yeah, revisit. Should be able to do that just fine...